internal/slicer/slicer.go

```diff
@@ -196,14 +201,17 @@ func Run(options *RunOptions) (*Report, error) {
 	// Create new content not coming from packages.
 	done := make(map[string]bool)
 	for _, slice := range options.Selection.Slices {
-		arch := archives[slice.Package].Options().Arch
+		arch := pkgArchives[slice.Package].Options().Arch
 		for targetPath, pathInfo := range slice.Contents {
 			if len(pathInfo.Arch) > 0 && !contains(pathInfo.Arch, arch) {
 				continue
 			}
-			if done[targetPath] || pathInfo.Kind == setup.CopyPath || pathInfo.Kind == setup.GlobPath {
+			if pathInfo.Kind == setup.CopyPath || pathInfo.Kind == setup.GlobPath || pathInfo.Kind == setup.GeneratePath {
 				continue
 			}
+			if done[targetPath] {
+				report.Entries[targetPath].Slices[slice] = true
```

Comment:
We need a comment here stating that the content created would have been the same and that is why we are only marking the new slice and not doing anything else.

Reply:
Added: ad54c29